### ISSUE TITLE
Review the use of aria-label on a <ul> element in the site navigation menu

### ISSUE THREAD

**Issue by domoscargin** [alphagov/govuk-design-system#2624]
## What
We use an `aria-label` on our sub-navigation list `<ul>`. This causes HTML validation issues, so we need to make the call on whether we keep it as an exception, add an explicit `aria-role` to keep validation happy, or get rid of it and find another solution.

## Why
Previously, we added an `aria-label` to the main site nav's Community sub-menu to make a better experience for screenreaders:  https://github.com/alphagov/govuk-design-system/commit/4ff06e4961bcd4a147d580d1d5ac2736fe9f6ec3

Recently, we [added HTML linting](https://github.com/alphagov/govuk-design-system/pull/2569), and it flagged this as `aria-label-misuse`.

If we assign an explicit `role` to the labelled list, the HTML validator is sated and complains no longer.

However, it's a bit unclear which `role` we'd want to add - `navigation` is out since this subnav is already contained within a `nav` parent, and `menu` [might not be right for this situation](https://github.com/alphagov/govuk-design-system/pull/2569#discussion_r1115422380).

We could avoid this debate if we just find a solution that doesn't involve ARIA. First step would probably be to just remove the `aria-label` and test whether it reads fine in multiple screenreaders.

If it’s missing context without the aria-label, we could look at using a bit of ‘visually-hidden’ instead of an aria-label. Would need to test to see if it works consistently across all screen readers though.

## Who needs to work on this
Developer, Accessibility specialist(?)

## Who needs to review this
Developer, Accessibility specialist

## Done when

- [ ] We have a solution that is both valid HTML, and a good experience for screenreaders.
- [ ] We have implemented that solution (this will likely involve [removing the inline validation exception](https://github.com/alphagov/govuk-design-system/blob/9ba07459f46a6e9738067c7ec90da6dd22f3a1b6/views/partials/_navigation.njk#L14))

**Comment by 36degrees** [comment]
Relevant blog post on labelling lists that looks like it'd be worth a read: https://www.scottohara.me/blog/2020/05/02/labelled-lists.html

**Comment by 36degrees** [comment]
The new navigation introduced in #4701 does not include an `aria-label` as [the old one did](https://github.com/alphagov/govuk-design-system/blob/9ab7f49286c1f43f50c5907b3449bb03e0e50295/views/partials/_navigation.njk#L14-L15), so I reckon we can close this now.

**Comment by romaricpascal** [comment]
> The new navigation introduced in [#4701](https://github.com/alphagov/govuk-design-system/pull/4701) does not include an `aria-label` as [the old one did](https://github.com/alphagov/govuk-design-system/blob/9ab7f49286c1f43f50c5907b3449bb03e0e50295/views/partials/_navigation.njk#L14-L15), so I reckon we can close this now.

The `aria-label` was missed when replicating the accordion, so its removal is not intentional.

In terms of effect, users get a slightly different announcement when entering the first item of the sub-navigation list.
Comparing with the [v4 backup of the Design System site](https://v4--govuk-design-system-preview.netlify.app/), which had the same mobile navigation as before #4701, and using VoiceOver on Safari:
- before, with the `aria-label`, they'd get "Patterns overview, list **Patterns** 4 items, level 2"
- now, without the `aria-label`, they only get "Patterns overview, list 4 items, level 2"

I've tracked the labelling a bit further back to https://github.com/alphagov/govuk-design-system/commit/b83b3e77682378757aa6a69f5eb78235180cde1c, which was using an `aria-labelledby` at the time (since replaced in https://github.com/alphagov/govuk-design-system/commit/4ff06e4961bcd4a147d580d1d5ac2736fe9f6ec3, as @domoscargin noted in the issue description).

I'd say we may still want to review that `aria-label`, except to decide whether necessary to re-introduce it rather than remove it, so I'll reopen the issue.
